Formula/signal-cli.rb

```diff
@@ -23,9 +23,9 @@ class SignalCli < Formula
   resource "libsignal-client" do
     # per https://github.com/AsamK/signal-cli/wiki/Provide-native-lib-for-libsignal#libsignal-client
     # we want the specific libsignal-client version from 'signal-cli-0.8.1/lib/signal-client-XXXX-X.X.X.jar'
-    version "java-0.2.3"
+    version "v0.8.1"
     url "https://github.com/signalapp/libsignal-client/archive/refs/tags/#{version}.tar.gz"
```

Comment:
New version format needs to causing issue for JAR filename in zip step
Filename: `signal-client-java-0.8.1.jar`
Below format: `libexec/"lib/signal-client-#{r.version}.jar"`

Reply:
I did not see that jar in the lib though.
```
$ ls signal-cli-0.8.4.1/lib/signal-*
signal-cli-0.8.4.1/lib/signal-cli-0.8.4.1.jar
signal-cli-0.8.4.1/lib/signal-client-java-0.8.1.jar
signal-cli-0.8.4.1/lib/signal-service-java-2.15.3_unofficial_24.jar
```

Reply:
I will raise in the upstream repo.

Reply:
I was thinking something like:

```suggestion
    version "0.8.1"
    url "https://github.com/signalapp/libsignal-client/archive/refs/tags/v#{version}.tar.gz"
```

And:
```diff
-      system "zip", "-d", libexec/"lib/signal-client-#{r.version}.jar", "libsignal_jni.so"
+      system "zip", "-d", libexec/"lib/signal-client-java-#{r.version}.jar", "libsignal_jni.so"
```

Reply:
@chenrui333 this isn't an upstream issue. Please avoid opening issues upstream for build failures when the problem is in the formula.

Reply:
> @chenrui333 this isn't an upstream issue. Please avoid opening issues upstream for build failures when the problem is in the formula.

I dont think there is any problem asking about upstream's feedback when we dont need to change anything. (in this case, it would be the filename change). Maybe the `upstream issue` tag is kind of misleading, I will not add that tag in the future.

Reply:
It's not that. I don't think we want to waste upstream maintainer time when the problem is actually in the formula.